packages/eslint-plugin-khan/test/rules/aphrodite-add-style-variable-name.test.ts

```diff
@@ -0,0 +1,120 @@
+import {RuleTester} from "@typescript-eslint/rule-tester";
+
+import {rules} from "../../src/index";
+
+const ruleTester = new RuleTester({
+    languageOptions: {
+        parserOptions: {
+            ecmaVersion: 6,
+            sourceType: "module",
+            ecmaFeatures: {},
+        },
+    },
+    linterOptions: {
+        // NOTE(kevinb): Avoids 'TypeError: Expected a Boolean' error
+        // when running the tests.
+        reportUnusedDisableDirectives: true,
+    },
+});
+
+const ruleName = "aphrodite-add-style-variable-name";
+const rule = rules[ruleName];
+
+ruleTester.run(ruleName, rule, {
+    valid: [
+        {
+            code: `const StyledDiv = addStyle("div")`,
+        },
```

Comment:
Does this rule ignore things like `const MyCompWithStyle = addStyle(MyComp)`?

Reply:
Good question, yes! It will return early if the addStyle argument is not a string:

https://github.com/Khan/wonder-stuff/blob/fc49725b62cbb3fb7cfb179c5eb71e6d7f3fe323/packages/eslint-plugin-khan/src/rules/aphrodite-add-style-variable-name.ts#L79-L85

I updated the tests and docs to explain that the naming convention only applies to string arguments for html tags!

Reply:
I don't see any uses of it in our code.  I usually tell people to manually add a `style` prop.  Since that's what all of our code does I guess people just copy/pasting that now which is good.